docs/content/guides/author-apps/aws/howto-aws-resources/index.md

```diff
@@ -52,15 +52,21 @@ Follow the prompts to install the [control plane services]({{< ref "/concepts/te
 - **Add AWS provider** - An [AWS cloud provider]({{< ref "/guides/operations/providers/aws-provider" >}}) allows you to deploy and manage AWS resources as part of your application. Follow the how-to guides to [configure the AWS provider]({{< ref "/guides/operations/providers/aws-provider/howto-aws-provider" >}}) with the preferred identity.
 - **Environment name** - The name of the environment to create. You can specify any name with lowercase letters, such as `myawsenv`.
 
-## Step 3: Create a Bicep file to model AWS Simple Storage Service (S3)
+## Step 3: Create a `bicepconfig.json` in your application's directory 
```

Comment:
As discussed lets add a step that says select "no " to setup application so that the steps remain the same.

Reply:
Updated